melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/sed/mapper/til_sed/lovvalg/LovvalgSedMapper.java

```diff
@@ -17,18 +17,18 @@
     // bør denne metoden overrides.
     @Override
     default SED mapTilSed(SedDataDto sedData) {
-        SED sed = SedMapper.super.mapTilSed(sedData);
+        var sed = SedMapper.super.mapTilSed(sedData);
         sed.setMedlemskap(getMedlemskap(sedData));
 
         return sed;
     }
 
     default void setVedtaksdata(Vedtak vedtak, VedtakDto vedtakDto) {
-        if (vedtakDto != null) {
-            if (!vedtakDto.isErFørstegangsvedtak()) {
-                vedtak.setEropprinneligvedtak("nei");
-                vedtak.setDatoforrigevedtak(vedtakDto.getDatoForrigeVedtak().toString());
-            }
+        if (vedtakDto != null && !vedtakDto.isErFørstegangsvedtak()) {
+            vedtak.setEropprinneligvedtak("nei");
+            vedtak.setDatoforrigevedtak(
+                    vedtakDto.getDatoForrigeVedtak() != null ? vedtakDto.getDatoForrigeVedtak().toString() : null
```

Comment:
Hvorfor skjer dette?
"ukjent" i stedet for null?

Reply:
Blir litt usikker etter jeg opprettet PRen, for det bør jo ikke skje 👀  
Ser kun feilen i Q2, men ikke noe i prod.. Er det kanskje bare noen stygge behandlinger i Q2?

Reply:
> ja, SEDen burde strengt talt ta i mot LocalDate som felt

Enig, `LocalDate` ville være bedre enn `String`.